common/reviews/api/rush-lib.api.ts

```diff
@@ -92,6 +92,15 @@ enum ChangeType {
   patch = 2
 }
 
+// @public
+class CommandLineConfiguration {
+  // (undocumented)
+  public commands: Map<string, ICustomCommand>;
+  // (undocumented)
+  public options: Map<string, ICustomOption>;
+  public static tryLoadFromFile(jsonFilename: string): CommandLineConfiguration;
+}
```

Comment:
Don't expose this.  @iclanton is supposed to be working to trim down the API to a minimal set of configuration parsing functionality, now that we eliminated the rush/rush-lib boundary